### DISCUSSION TITLE
Use VDA5050 2.0

### DISCUSSION THREAD

**Discussion by HardyBenjamin** [openTCS/opentcs#92] [Q&A]
Hello,
Once the application and the VDA5050 extension are downloaded and extracted, how do I implement the extension into the application? (Step by step, in the most comprehensive and easiest way)

Thank you.

**Comment by swltr** [comment]
The current version of the [openTCS example integration project](https://github.com/openTCS/opentcs-integration-example) integrates the VDA5050 driver (see https://github.com/openTCS/opentcs-integration-example/commit/539eaa0108d454115dd52721ee97a051890a37b4 for the commit that added it). The [Developer's Guide](https://opentcs.org/docs/5.9/developer/developers-guide/opentcs-developers-guide.html#_generating_an_integration_project) describes how to set up your own openTCS integration project, which is the recommended way to start your own openTCS-based project.
